dataprofiler/tests/data/csv/daily-activity-sheet-@.csv

```diff
@@ -0,0 +1,32 @@
+COVID19 Activity Sheet
+Time@Activity @Supplies@Notes
+06:00am - 07:30am@Breakfast / Get Ready@Food@
+07:30am - 07:45am @Potty + Diapers@Potty@Non-primary parent
+08:00am - 09:00am@Structured Play@"Books, Song Books, Music"@
+09:00am - 09:30am@Snack Time@Food@
+09:30am - 10:00am @Craft@"Draw, Paint, Crafts, etc."@At Kitchen Table
+10:00am - 10:15am@Potty + Diapers@Potty@Primary parent
+10:15am - 11:30am @Free Play@Outside or Inside@Only one station at a time
+11:30am - 12:00pm@Lunch@Food@
+12:00pm - 12:15pm@Potty + Diapers@Potty@Both parents Split
+12:15pm - 02:30pm@Nap@@
+02:00pm - 03:00pm@Potty + Diapers@Potty@Primary parent
+03:00pm - 03:30pm@Snack Time@Food@
+03:30pm - 04:00pm@Craft or Game@"Playdough, Board game, etc."@
+04:00pm - 05:00pm@Free Play@Inside@
+05:00pm - 05:30pm@Dinner@Food@
+05:30pm - 05:45pm@Potty + Diapers@Potty@Both parents split
+05:45pm - 07:00pm@Bath(s) + Book & Bed @Bathtub + Books + Bed@
+07:00pm - 07:30pm@Books + Bed @Books + Bed@
+@@@
+Master@Stationary Activiy@Structured Play@

```

Comment:
imo, this looks like the second part of the csv file. Should we consider this example?

Reply:
and, when we take the first 5 lines for the detection, this won't affect the results. Did you increase 5 to 50 to see the difference?